components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/impl/APIGatewayPublisherImpl.java

```diff
@@ -310,4 +310,25 @@ private void saveEndpointConfig(String gwHome, String content) {
             }
         }
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override public void changeAPIState(API api, String status) throws GatewayException {
+
```

Comment:
Override annotation should be in a separate line and no need of extra line before first statement in the method